engine/src/main/java/org/terasology/engine/world/block/items/BlockItemComponent.java

```diff
@@ -21,4 +21,9 @@ public BlockItemComponent() {
     public BlockItemComponent(BlockFamily blockFamily) {
         this.blockFamily = blockFamily;
     }
+
+    @Override
+    public void copy(BlockItemComponent other) {
+        this.blockFamily = other.blockFamily;
```

Comment:
Do we need to deep-copy `BlockFamily`?

Reply:
I don't think you can modify a BlockFamily. its treated as a singleton.